Description: Unbreak XInclude of SGCheck-specific options in manpage.
 The list that appears to be the intended target of the XInclude is
 currently commented out (presumably because it looks silly with no
 entries), so give the relevant id to the paragraph saying there *are*
 no SGCheck-specific options.
Origin: vendor
Author: Samuel Bronson <naesten@gmail.com>
Bug-Debian: http://bugs.debian.org/696694
Last-Update: 2013-01-06

--- valgrind-3.8.1.orig/exp-sgcheck/docs/sg-manual.xml
+++ valgrind-3.8.1/exp-sgcheck/docs/sg-manual.xml
@@ -31,7 +31,7 @@ observation about the likely forms of st
 <sect1 id="sg-manual.options" xreflabel="SGCheck Command-line Options">
 <title>SGCheck Command-line Options</title>
 
-<para>There are no SGCheck-specific command-line options at present.</para>
+<para id="sg.opts.list">There are no SGCheck-specific command-line options at present.</para>
 <!--
 <para>SGCheck-specific command-line options are:</para>
 
